src/single-spa-angular1.js

```diff
@@ -68,7 +72,9 @@ function mount() {
 
 function unmount() {
 	return new Promise((resolve, reject) => {
-		let rootScope = angular.injector(['ng']).get('$rootScope');
+		let rootElement = angular.element(getContainerEl().querySelector(`#${SINGLE_SPA_ELEMENT_ID}`));
```

Comment:
Not using `getElementById` because it isn't available at a node level and I don't want to use `document.getElementById` because there might be more than one instance of this element.